routes/sessions/session.js

```diff
@@ -69,7 +69,7 @@ module.exports = () => {
   // POST /logout - Log user out by removing encrypted cookie
```

Comment:
I don't understand why this change is necessary. Can you please explain!

Reply:
sure, this is blah blah